### DISCUSSION TITLE
Network Transform world vs local sync

### DISCUSSION THREAD

**Discussion by vojtechmusilek** [FirstGearGames/FishNet#212] [Feature Request]
Hi, this is not really feature request, it is more like idea which I implemented and could possibly be merged as new feature.

I have use case where synchronizing world positions instead of local position makes the code so much cleaner.
I prototyped synchronization of 'position' instead of 'localPosition' and it looks like it is working.

Question is why is 'localPosition' used?
Is that just because of the data saving or is there something I don't see?

The same applies for 'localRotation' vs 'rotation'

**Comment by FirstGearGames** [comment]
Local is used to support nested objects. There are no plans to switch to world space.
Adding both would create convoluted code for little reason.
